theano/gpuarray/subtensor.py

```diff
@@ -389,7 +389,7 @@ def add_to_zview(self, nodename, x, fail):
           void *args[2];
           args[0] = &zview->ga;
           args[1] = &%(x)s->ga;
-          if (GpuElemwise_call(iadd, args, GE_BROADCAST) != GA_NO_ERROR) {
+          if (GpuElemwise_call(iadd, args, GE_BROADCAST | GE_PADSHAPE) != GA_NO_ERROR) {
```

Comment:
The perform() doesn't need to be fixed since the python interface will set this flag to True by default.